book/src/developers/contributing/releases/deployment.md

```diff
@@ -1,37 +1,100 @@
+### Run ansible
+- Check monitoring tools to understand network health, and compare against post-deployment, eg~
+    - [Glados](http://glados.ethportal.net/content/)
+- Go into Portal section of Ansible: `cd portal-network/trin/ansible/`
+- Run the deployment: `ansible-playbook playbook.yml --tags trin`
+- Wait for completion
+- Launch a fresh trin node, check it against the bootnodes
+- ssh into a random node and a random bridge node, to check the logs:
+	- [find an IP address](https://github.com/ethereum/cluster/blob/master/portal-network/trin/ansible/inventories/dev/inventory.yml)
+	- `ssh ubuntu@$IP_ADDR`
+  - check logs, ignoring DEBUG: `sudo docker logs trin -n 1000 | grep -v DEBUG`
+- Check monitoring tools to see if network health is the same or better as before deployment. Glados might lag for 10-15 minutes, so keep checking back.
+- ?? Also release glados, to use the latest trin ??
 
-As trin stabilizes, more notifications will be necessary (twitter, blog post, etc).
+### Communicate
 
-## Update these docs
+Notify in Discord chat about the network nodes being updated.
+
+### Update these docs
 
 Immediately after a release is the best time to improve these docs:
 - add a line of example code
 - fix a typo
 - add a warning about a common mistake
 - etc.
 
-The source for this section is at `book/src/developers/contributing/releases/`.
 For more about generally working with mdbook see the guide to [Contribute to
-our book](/developers/contributing/book.md).
+the book](/developers/contributing/book.md).
 
-## Celebrate
+### Celebrate
 
 Another successful release! 🎉
+
+## FAQ
+### Key Failures
+You might see this when running ansible, or the sops check:
+```shell=
+Failed to get the data key required to decrypt the SOPS file.
+
+Group 0: FAILED
+  32F602D86B61912D7367607E6D285A1D2652C16B: FAILED
+    - | could not decrypt data key with PGP key:
+      | github.com/ProtonMail/go-crypto/openpgp error: Could not
+      | load secring: open ~/.gnupg/secring.gpg: no such
+      | file or directory; GPG binary error: exit status 2
+
+  81550B6FE9BC474CA9FA7347E07CEA3BE5D5AB60: FAILED
+    - | could not decrypt data key with PGP key:
+      | github.com/ProtonMail/go-crypto/openpgp error: Could not
+      | load secring: open ~/.gnupg/secring.gpg: no such
+      | file or directory; GPG binary error: exit status 2
+
+Recovery failed because no master key was able to decrypt the file. In
+order for SOPS to recover the file, at least one key has to be successful,
+but none were.
+```
+It means your key isn't working. Check with `@paulj`.
+### Deployment Warnings
+You might see this during a deployment:
+> fatal: [trin-ams3-18]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 178.128.253.26 port 22: Connection timed out", "unreachable": true}
+
+Retry once more. If it times out again, ask `@paulj` to reboot the machine.
```

Comment:
Ooooh, nice! Did this actually get resolved? I'm quite familiar with that machine always failing and typically just let it fail since it was the only one that failed, so i'm glad to see that it was resolved!

Reply:
Well, "resolved" as in he rebooted it, and the next deploy succeeded.  But, maybe it will fail again at the next deploy? 🤷🏻‍♂️